docs/develop/dotnet/failure-detection.mdx

```diff
@@ -125,6 +125,21 @@ return await Workflow.ExecuteActivityAsync(
     });
 ```
 
+### Overriding the retry interval with Next Retry Delay {#next-retry-delay}
+
+You may throw an [Application Failure](/references/failures#application-failure) with the NextRetryDelay field set. This value will replace and override whatever the retry interval would be on the retry policy.
```

Comment:
```suggestion
Throw an [Application Failure](/references/failures#application-failure) with an assigned `NextRetryDelay` field to replace and override the Retry interval defined in the Retry Policy.
```

Reply:
This is the form already present/approved at `docs/develop/java/failure-detection.mdx` and may be a rule worth applying consistently there too instead of just here (but feel free to apply here first)